test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -555,7 +555,7 @@ public class CompatibilityTests
         [InlineData("netstandardapp1.5", "win8", false)]
 
         // netcoreapp only supports netstandard
-        [InlineData("netcoreapp9.0", "netstandard2.1", false)]
+        [InlineData("netcoreapp9.0", "netstandard2.1", true)]
```

Comment:
The pattern here is to have a negative test for the next version. 

So you should add a test case like the below: 

>         [InlineData("netcoreapp9.0", "netstandard2.2", false)]

Reply:
Thanks! I'll add the line for netstandard2.2.